changelogs/fragments/redhat_subscribtion.yml

```
@@ -0,0 +1,5 @@
+bugfixes:
+  - made code compatible with Python3
```

Since these changelog fragments get [rolled up into one file](https://github.com/ansible/ansible/blob/stable-2.8/changelogs/CHANGELOG-v2.8.rst#v2-8-3), it is helpful to put the module name in the entry as well as link to the issue PR to provide further context.
```suggestion
  - redhat_subscription - made code compatible with Python3 (https://github.com/ansible/ansible/pull/58665)
```

---

Thanks! Done.